libmattermost.c

```diff
@@ -4821,6 +4746,16 @@ mm_add_buddy(PurpleConnection *pc, PurpleBuddy *buddy, PurpleGroup *group, const
 	const gchar *buddy_name = purple_buddy_get_name(buddy);
 	const gchar *user_id = g_hash_table_lookup(ma->usernames_to_ids, buddy_name);
 	
+	if (purple_strequal(user_id,ma->self_user_id)) {

```

Comment:
Not sure why hiding yourself on your own buddy list is part of a PR about multi-teams

Also, I add myself onto my own list to be able to see what my status is, so having it be removed is a pain in the butt.